modules/spring-boot-client-autoconfigure/examples/main/resources/application.yml

```diff
@@ -0,0 +1,4 @@
+ignite-client:
```

Comment:
Look at the comment about the config above.

Reply:
I think we should go with the simplest prefix. Is it OK for you?